src/Plugin_Command.php

```diff
@@ -828,7 +831,33 @@ public function get( $args, $assoc_args ) {
 	 */
 	public function uninstall( $args, $assoc_args = array() ) {
 		$successes = $errors = 0;
+
+		$all = Utils\get_flag_value( $assoc_args, 'all', false );
+
```

Comment:
Could you do
```
		if ( ! ( $args = $this->check_optional_args_and_all( $args, $all ) ) ) {
			return;
		}
```
here. This will mean the `if ( $all ) {` block isn't needed.